xml/containers-support.xml

```diff
@@ -12,47 +12,63 @@
+      <imagedata fileref="container_support_matrix.png" width="100%" format="PNG"/>
+     </imageobject>
+     <imageobject role="html">
+      <imagedata fileref="container_support_matrix.png" width="100%" format="PNG"/>
+     </imageobject>
+    </mediaobject>
+   </informalfigure>
+   <para>
+    In most scenarios, all &slea; containers are expected to be interoperable if
+    the application or its dependencies do not interact directly with kernel
+    version-specific data structures (<literal>ioctl</literal>,
+    <literal>/proc</literal>, <literal>/sys</literal>,
+    <literal>routing</literal>, <literal>iptables</literal>,
+    <literal>nftables</literal>, <literal>eBPF</literal>, etc.) or modules
+    (KVM, OVS, SystemTap, etc.). Support for <literal>ioctl</literal> and
+    access to <literal>/proc</literal> is limited to the most common scenarios
+    needed by unprivileged users.
+   </para>
+  </sect2>
 
-  <para>
-   In most scenarios, all SLE containers are expected to be interoperable if
-   the application or its dependencies do not interact directly with kernel
-   version-specific data structures (<literal>ioctl</literal>,
-   <literal>/proc</literal>, <literal>/sys</literal>,
-   <literal>routing</literal>, <literal>iptables</literal>,
-   <literal>nftables</literal>, <literal>eBPF</literal>, etc.) or modules (KVM,
-   OVS, SystemTap, etc.). Support for <literal>ioctl</literal> and access to
-   <literal>/proc</literal> is limited to the most common scenarios needed by
-   unprivileged uses.
-  </para>
-  <important>
-  <title>&slea; BCI support for managed Kubernetes services in the public cloud</title>
-  <para>
-  The support status of a &slea; Base Container Image (BCI) is determined by the
-  runtime used to run containers. &slea; Base Containers are supported when used
-  with containerd, CRI-O and &docker;, and &podman;. Container images based on
-  &sle; 12 SP5 through &sle; 15 SP4 are fully supported on Amazon Elastic
-  Container Service for Kubernetes. Although Microsoft Azure Kubernetes Service,
-  Google Kubernetes Engine, and Alibaba Container Service for Kubernetes provide
-  runtimes that can be used for running BCIs, they haven't been fully tested.
-  </para>
-  </important>
+  <sect2 xml:id="sec-sle-container-sle-host-support">
+   <title>Support for non-&slsa; hosts</title>
+   <para>
+    While SUSE-based containers are fully supported, issues in the host
+    environment must be handled by the host environment vendor. SUSE supports
+    components that are part of the SUSE base containers. Packages from SUSE
+    repositories are also supported. Additional components and applications in
+    the containers are not covered by SUSE support. A &sle; subscription is
+    required for building derived containers.
+   </para>
+   <para>
+   Containers based on &sls; 12 SP5 and &sls; 15 (all Service Packs) are
+   supported according to their official lifecycles and the following table.
```

Comment:
Above you use the term "matrix", now this is a "table". Better use the same term for both images.